rmgpy/solver/liquidTest.py

```diff
@@ -53,36 +53,56 @@ class LiquidReactorCheck(unittest.TestCase):
 
     def setUp(self):
         """
-        Here we choose a kinetic model consisting of the hydrogen abstraction reaction
-        CH4 + C2H5 <=> CH3 + C2H6.
-
+        Here we choose a kinetic model consisting of the hydrogen abstraction
```

Comment:
This is ok, but in general don't make these sorts of changes because while it's better to have code more compliant with PEP8 they're difficult to review (I have to go through and guarantee that the code will run the same as before), usually a waste of your time and can in other cases crowd out important changes in a commit log.  It's very good to keep your changes and the close vicinity around them PEP8 compliant, but don't go overboard with it.

Reply:
To add on, updates to code style are always appreciated, but if you plan on making a lot of style changes, it is helpful to put them in a separate commit to make reviewing easier (i.e. commit with style changes only, then commit with functional changes).

Reply:
Ok, thanks for the advice!